app/server_test.go

```diff
@@ -329,9 +329,6 @@ func TestPanicLog(t *testing.T) {
 	client := &http.Client{Transport: tr}
 	client.Get("https://localhost:" + strconv.Itoa(s.ListenAddr.Port) + "/panic")
 
-	err = s.Shutdown()
```

Comment:
Was it intentional to not shutdown server as part of this test?  When we switch to async logging this year we'll need shutdown to flush logs.

Reply:
This is totally unintentional, thanks for catching this. My intent was to remove error check since this method never returns an error.